charts/eks/values.yaml

```diff
@@ -208,7 +208,7 @@ api:
 coredns:
   image: public.ecr.aws/eks-distro/coredns/coredns:v1.8.4-eks-1-21-8
   replicas: 1
-
+  resources:
```

Comment:
While this works, please put an empty string here instead as it is more readable